src/script_opt/ZAM/Driver.cc

```diff
@@ -47,13 +47,11 @@ void ZAMCompiler::Init() {
 }
 
 void ZAMCompiler::InitGlobals() {
-    for ( auto g : pf->Globals() ) {
-        auto non_const_g = const_cast<ID*>(g);
-
+    for ( auto& g : pf->Globals() ) {
         GlobalInfo info;
```

Comment:
```suggestion
        GlobalInfo info{.id = g, .slot = AddToFrame(g)};
```

And then delete the two lines below.

Reply:
Nifty. I'm guessing this is new w/ C++20. I've pushed that change, and also the rebase (thanks for the help with that!).